src/utils/tools/get-lab-config.sh

```diff
@@ -1,10 +1,7 @@
 #!/usr/bin/env bash
 
-USER="$1"
-TOKEN="$2"
-
 if [[ "$USER" == "" || "$TOKEN" == "" ]]; then
-    echo "usage: $0 STUD_X GIT_TOKEN"
+    echo "USER and TOKEN environment variables not set."
```

Comment:
Unde se setează astea doua env vars?

Reply:
In `_K8s lab - init` vor fi informatiile.